src/main/java/org/marek/model/User.java

```diff
@@ -0,0 +1,18 @@
+package org.marek.model;
+
+import javax.persistence.Entity;
+
+@Entity
+
```

Comment:
Anotace `@Entity` patří ke třídě `User`, takže musí být nad tím, řádek navíc mezi tím to dělá nečitelným.